routes/items.js

```diff
@@ -0,0 +1,72 @@
+/*eslint-env node*/
+const express = require('express');
+const router = express.Router();
+const alasql = require('alasql');
+require('../item_db.js');
+require('../config/auth.js');
+const middlewares = require('../config/middlewares.js');
+const bunyan = require('bunyan');
+
+router.param('itemId', middlewares.itemId);
+const log = bunyan.createLogger({name: 'items'});
+
+// ----- Get all Items -----
+router.get('/items/', middlewares.loggedIn, (req, res, next) => {
+	const items = alasql('SELECT * FROM items WHERE userId=?', req.user.id);
+	log.info(items);
+	res.send(items);
+	log.info(items + 'Items found for user '+ req.user.username);
+	next();
+});
+
+// ----- Get a single Item -----
+router.get('/items/:itemId', middlewares.loggedIn, (req, res, next) => {
+	const userId = alasql('SELECT userId FROM items WHERE itemId='+ req.itemId);
```

Comment:
this might be calling `getUserById` from `/src/model/user.js`
etc, everywhere below.